Cargo.toml

```diff
@@ -36,53 +36,62 @@ product-name = "Space Acres"
 
 [dependencies]
 anyhow = "1.0.82"
-image = { version = "0.25.1", default-features = false, features = ["png"] }
-betrayer = { version = "0.2.0" }
 arc-swap = "1.7.1"
 async-lock = "3.3.0"
+async-oneshot = "0.5.9"
 async-trait = "0.1.80"
 backoff = { version = "0.4.0", features = ["futures", "tokio"] }
+betrayer = { version = "0.2.0" }
 bytesize = "1.3.0"
 clap = { version = "4.5.4", features = ["derive"] }
 dark-light = "1.1.1"
 dirs = "5.0.1"
 duct = "0.13.7"
 event-listener-primitives = "2.0.1"
-indoc = "2.0.5"
 file-rotate = "0.7.5"
 frame-system = { git = "https://github.com/subspace/polkadot-sdk", rev = "44d742b90e7852aed1f08ab5299d5d88cfa1c6ed", default-features = false }
 fs4 = "0.8.2"
 futures = "0.3.30"
+futures-timer = "3.0.3"
 gtk = { version = "0.7.3", package = "gtk4" }
 hex = "0.4.3"
+image = { version = "0.25.1", default-features = false, features = ["png"] }
+indoc = "2.0.5"
 # Substrate uses old version of libp2p
 libp2p-identity-substate = { version = "0.1.3", package = "libp2p-identity" }
+lru = "0.12.3"
```

Comment:
Might want to switch to a different library right away, see https://github.com/subspace/subspace/pull/2736 for corresponding effort on monorepo side